datascience/tables.py

```diff
@@ -3066,11 +3072,20 @@ def __getitem__(self, row_indices_or_slice):
         A+           | 4
         A            | 4
         A-           | 3.7
+        >>> grades.take(0, 2)
+        letter grade | gpa
+        A+           | 4
+        A-           | 3.7
         >>> grades.take(10)
         Traceback (most recent call last):
             ...
         IndexError: index 10 is out of bounds for axis 0 with size 6
         """
+        # print(row_indices_or_slice)
```

Comment:
These look like remnants of personal debugging, and should be removed before merging.